website/content/docs/getting-started-troubleshooting.md

```diff
@@ -63,6 +63,44 @@ INFO: Elapsed time: 1.883s.
 
     Then fix it correspondingly.
     
+* It is also possible that the host has an issue with resolving localhost.
+To check, run the following command in a shell. 
+    
```

Comment:
Can u also add some sample errors output, so people can associate their errors with this trouble-shooting?